find_line_in_image.py

```diff
@@ -0,0 +1,72 @@
+# the modules we need for finding the distance and angle to the line
+import cv2
+import numpy as np
+import sys
+
+
+def find_line_contour(image):
+    opened_threshold_image = __clean_image(image)

```

Comment:
__clean_image function does not return any value

Reply:
that is right i should fix that